Bottom: 68e3ff1209974573f9363b913206bba39ec8f0dc
Top:    1380fb77d4c96a8a26b5425250074006ba4e0c2e
Author: Igor Stoppa <igor.stoppa@huawei.com>
Date:   2018-06-07 16:06:51 +0300

Use designated initializer for struct list_head in list.h


---

diff --git a/include/linux/list.h b/include/linux/list.h
index 4b129df4d46b..11847c96571f 100644
--- a/include/linux/list.h
+++ b/include/linux/list.h
@@ -18,7 +18,10 @@
  * using the generic single-entry routines.
  */
 
-#define LIST_HEAD_INIT(name) { &(name), &(name) }
+#define LIST_HEAD_INIT(name) {	\
+	.next = &(name),	\
+	.prev = &(name),	\
+}
 
 #define LIST_HEAD(name) \
 	struct list_head name = LIST_HEAD_INIT(name)
